client/src/actions/user.js

```diff
@@ -35,6 +35,7 @@ export function getUserInfo () {
             .get('http://localhost:8000/app/users/me/')
             .set('Authorization', 'Token ' + localStorage.getItem('token'))
             .end(function(err, res) {
+                // NOTE don't think '404' works
```

Comment:
No this doesn't work because you're comparing a number (res.status) against a string, it will never be true.